content/en/docs/tasks/observability/metrics/customize-metrics/index.md

```diff
@@ -100,21 +100,6 @@ dimensions.
     $ kubectl -n istio-system apply -f stats-filter-1.6.yaml
     {{< /text >}}
 
-1. Apply the following annotation to all injected pods with the list of the
-   dimensions to extract into a Prometheus
-   [time series](https://en.wikipedia.org/wiki/Time_series), using the following
-   command:
-
-    {{< text yaml >}}
-    apiVersion: extensions/v1beta1
-    kind: Deployment
-    spec:
-    template:
-        metadata:
-        annotations:
-            sidecar.istio.io/extraStatTags: destination_port,request_host
```

Comment:
is this not needed anymore?
apart from this lgtm.

Reply:
we see the tags in prometheus without this and with this, there is a crash... thus, removing it..

Reply:
this is ok for `destination_port` and a few others, but isn't this required for anything non-standard ?  how do we want to handle that?

Reply:
@douglas-reid I'm not sure I understand if this comment affects the merge-ability of this PR or not? Are you suggesting a change might be needed in this PR, or is this one good to go as is?

Reply:
@douglas-reid : do we know the attributes for whom this is not needed? 
or I can add it in a note that. if you are using it for anything non-standard, then you might want to add this?

Reply:
i think this is needed for anything not in this list: https://github.com/istio/istio/blob/c24438d1706e4db7ffdacd2f686223f7a6c104c0/pkg/bootstrap/config.go#L188

Reply:
NOTE: I have not verified.

Reply:
Thanks Doug.. this looks right.. let me add this link and not delete the extraStatTags para...

Reply:
Fixed..PTAL

Reply:
Seems strange that the tip says it's only needed for things not it DefaultStatTags, but then the example shows two, both of which are in DefaultStatTags.

Anyway, @douglas-reid, do you think we should merge this now?

Reply:
Yes. Let's merge this now. We can improve the docs incrementally.